src/recordlinker/routes/patient_router.py

```diff
@@ -0,0 +1,66 @@
+"""
+recordlinker.routes.patient_router
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This module implements the patient router for the RecordLinker API. Exposing
+the patient API endpoints.
+"""
+
+import uuid
+
+import fastapi
+import sqlalchemy.orm as orm
+
+from recordlinker import schemas
+from recordlinker.database import get_session
+from recordlinker.linking import mpi_service as service
+
+router = fastapi.APIRouter()
+
+
+@router.post(
+    "/{patient_reference_id}/person",
+    summary="Assign Patient to new Person",
+    status_code=fastapi.status.HTTP_201_CREATED,
+)
+def create_person(
+    patient_reference_id: uuid.UUID, session: orm.Session = fastapi.Depends(get_session)
+) -> schemas.PatientPersonRef:
+    """
+    Create a new Person in the MPI database and link the Patient to them.
+    """
+    patient = service.get_patient_by_reference_id(session, patient_reference_id)
+    if patient is None:
+        raise fastapi.HTTPException(status_code=fastapi.status.HTTP_404_NOT_FOUND)
+
+    person = service.update_person_cluster(session, patient, commit=False)
```

Comment:
Since `commit` is `false` here when do the changes get committed?

Reply:
when the route is finished using the session.  https://github.com/CDCgov/RecordLinker/blob/main/src/recordlinker/database.py#L39